### ISSUE TITLE
Add Phalcon support

### ISSUE THREAD

**Issue by wajdijurry** [bartko-s/stefano-tree#13]
Could you please support Phalcon so we can use models instead of using direct database access ?

**Comment by bartko-s** [comment]
No. I never worked with Phalcon so I don't know how implement it properly. Next major release will be probably using only pure PDO adapter so I don't plan to support another framework.
